_posts/events/2024-08-09-pyopensci-fall-festival.md

```diff
@@ -38,7 +38,7 @@ Interested? Read on to learn more!
 * **Where:** Online using [Gather](https://www.gather.town/)
 * **When:** Monday, October 28--Friday, November 01, 2024
 * **Who:** If you are a scientist, a researcher, a student or anyone who is interested in making your workflows better, more open and reproducible, then this event is for you!
-* **Cost:** $300--$600, with scholarships and learner sponsorship opportunities
+* **Cost:** $350--$600, with scholarships and learner sponsorship opportunities
```

Comment:
@kierisi, please double-check these numbers.

Reply:
@lwasser from your feedback in the [Eventbrite Ticket task in asana](https://app.asana.com/0/1207924646051926/1207969249017423), the price is $350 for students. please let me know if that's changed.

Reply:
Eventbrite's upper range of ticket costs is 625, but it says 600 here. I found the same mismatch on the Eventbrite page text yesterday, but I fixed it there. Please let me know what I'm missing,

It's incredibly important that the costs are documented and the same in all of our copy. 

<img width="1654" alt="Screenshot 2024-09-10 at 5 08 15 PM" src="https://github.com/user-attachments/assets/0d9acda2-ea99-467b-b3fb-f9ea4ae8375b">

Reply:
```suggestion
* **Cost:** $350--$625, with scholarships and learner sponsorship opportunities
```
To make this simple let's just merge with this fix. please double-check the numbers and details of these posts. BUT also let me know if I'm missing some information associated with the actual range of costs.